apps/yapms/src/lib/components/modals/importmodal/ImportModal.svelte

```diff
@@ -71,12 +71,16 @@
 		loading = false;
 	}
 
-	function close() {
-		ImportModalStore.set({ ...$ImportModalStore, open: false });
-	}
+	const close =
+		$page.url.pathname === '/app/imported'
+			? () => {
+					goto('/');
+					$ImportModalStore.open = false;
+			  }
+			: undefined;
```

Comment:
undefined should be replaced with $ImportModalStore.open = false; here. Otherwise the modal won't close when you import a map (calls in the try catches wont run when undefined)